### PR TITLE
Web UI bootstrap

### PR DESCRIPTION
**Pull request by kepeket**
Prepare template override

### CODE REVIEW

File: web/ui/about.html.tpl [hunk 1]
```diff
@@ -0,0 +1,38 @@
+
+
+<html>
+  {{template "head"}}
+  <title>About <em>zoekt</em></title>
```
Comment (aladjadj) [inline web/ui/about.html.tpl]: can you explain these lines?
why the title is out of head

File: web/ui/about.html.tpl [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+<body>
+
+
```
Comment (aladjadj) [inline web/ui/about.html.tpl]: maybe remove whitelines

File: web/ui/didyoumean.html.tpl [hunk 3]
```diff
@@ -0,0 +1,9 @@
+
```
Comment (aladjadj) [inline web/ui/didyoumean.html.tpl]: maybe remove the whiteline

File: web/ui/head.html.tpl [hunk 4]
```diff
@@ -0,0 +1,43 @@
+
```
Comment (aladjadj) [inline web/ui/head.html.tpl]: maybe remove the whiteline

File: web/ui/jsdep.html.tpl [hunk 5]
```diff
@@ -0,0 +1,3 @@
+
```
Comment (aladjadj) [inline web/ui/jsdep.html.tpl]: maybe remove the whiteline?

File: web/ui/navbar.html.tpl [hunk 6]
```diff
@@ -0,0 +1,31 @@
+
```
Comment (aladjadj) [inline web/ui/navbar.html.tpl]: maybe remove the whiteline?

File: web/ui/print.html.tpl [hunk 7]
```diff
@@ -0,0 +1,23 @@
+
```
Comment (aladjadj) [inline web/ui/print.html.tpl]: maybe remove the whiteline?

File: web/ui/about.html.tpl (hunk 1 above)
Comment (aladjadj) [inline web/ui/about.html.tpl]: remove indent?

**Comment by kepeket** [comment]
@aladjadj It's an extract of the template produced by zoekt, they are untouched. It just to allow anyone to start to do things on templates simultaneously.

They are exported by zoekt, that's not a copy past, I've done 0 update on it yet.

I'll remove the bash script BTW

**Pull request closed without merging**